sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetScanBuilder.scala

```diff
@@ -49,10 +49,22 @@ case class ParquetScanBuilder(
     val pushDownStringStartWith = sqlConf.parquetFilterPushDownStringStartWith
     val pushDownInFilterThreshold = sqlConf.parquetFilterPushDownInFilterThreshold
     val isCaseSensitive = sqlConf.caseSensitiveAnalysis
+    val parquetOptions = new ParquetOptions(caseSensitiveMap, sparkSession.sessionState.conf)
+    val datetimeRebaseMode = DataSourceUtils.datetimeRebaseMode(
+      // Metadata of parquet files (spark version, for instance) is not available at the moment
+      _ => null,
+      parquetOptions.datetimeRebaseModeInRead)
     val parquetSchema =
       new SparkToParquetSchemaConverter(sparkSession.sessionState.conf).convert(readDataSchema())
-    val parquetFilters = new ParquetFilters(parquetSchema, pushDownDate, pushDownTimestamp,
-      pushDownDecimal, pushDownStringStartWith, pushDownInFilterThreshold, isCaseSensitive)
+    val parquetFilters = new ParquetFilters(
+      parquetSchema,
+      pushDownDate,
+      pushDownTimestamp,
+      pushDownDecimal,
+      pushDownStringStartWith,
+      pushDownInFilterThreshold,
+      isCaseSensitive,
+      datetimeRebaseMode)
     parquetFilters.convertibleFilters(this.filters).toArray
```

Comment:
We create `ParquetFilters` here only to check which filters are convertible, and rebase mode doesn't matter.

Shall we use pass `datetimeRebaseMode` as `CORRECTED` here?

Reply:
Yep. Let me pass `CORRECTED` here.